### ISSUE TITLE
Create a secure initial password

### ISSUE THREAD

**Issue by Idontker** [amosproj/amos2022ss04-digital-identity#128]
## User story

1. As a HR employee
2. I want / need a secure initial password
3. So that new HR employee can securly login after someone created their Account

## Acceptance criteria

- secure initial password is generated
- password is sent to new HR emplyoee via mail after a the registration succeded

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release

**Comment by Idontker** [comment]
Depends on #130
